config/boards/olimex-teres-a64.conf

```diff
@@ -0,0 +1,36 @@
+# OLIMEX Teres-A64
+# Maintainer: Jacob "Kreyren" Hrbek <kreyren@fsfe.org>
+
+# Optimized for Mainboard Rev.C
+
+BOARD_NAME="OLIMEX Teres A64" # Allwinner A64 quad core 2GB SoC Wi-Fi/BT
+
+# Specified in https://linux-sunxi.org/Allwinner_SoC_Family
+BOARDFAMILY="sun50iw1"
+
+# Has custom default configuration of the u-boot that should be used -- https://linux-sunxi.org/Olimex_Teres-A64#Mainline_U-Boot
+BOOTCONFIG="teres_i_defconfig"
+
+# Observed issues on insufficient size, thus enforced -- https://github.com/OLIMEX/DIY-LAPTOP/issues/52
+BOOTSIZE="256"
+
+# Seems to be the most reliable and bootloader should stay as separate partition
+BOOTFS_TYPE="fat"
+
+# Legacy has been removed as it's too old and disfunctional
+KERNEL_TARGET="current,edge"
+
+# gpt was not tested
+IMAGE_PARTITION_TABLE="msdos"
```

Comment:
I am almost sure this is not needed to be defined. Isn't this default?

Reply:
it's default, but gpt was not tested (as mentioned) so this is best practice for production environment and stable system.

Reply:
No reply for 4 days, considering stale